lib/components/src/Zoom/ZoomElement.tsx

```diff
@@ -24,15 +62,28 @@ export function ZoomElement({ scale, children }: ZoomProps) {
   const componentWrapperRef = useRef<HTMLDivElement>(null);
   const [height, setHeight] = useState(0);
 
+  const handleMutations = useCallback(() => {
+    setHeight(componentWrapperRef.current.getBoundingClientRect().height);
+  }, []);
+
   useEffect(() => {
     if (componentWrapperRef.current) {
       setHeight(componentWrapperRef.current.getBoundingClientRect().height);
     }
   }, [scale, componentWrapperRef.current]);
 
+  useMutationObserver({
+    element: componentWrapperRef,
+    options: { subtree: true, childList: true },
```

Comment:
`options` should be memoized else we'll have a new object with each render, which will cause the `useEffect` above to disconnect and re-observe.